lib/msf/core/payload/windows/x64/reflective_pe_loader.rb

```diff
@@ -6,7 +6,17 @@
 module Msf
   module Payload::Windows::ReflectivePELoader_x64
     include Payload::Windows::BlockApi_x64
-    def asm_reflective_pe_loader_x64
+    def asm_reflective_pe_loader_x64(opts)
+
+      prologue = ''
+      if opts[:is_dll] == true
+        prologue = %(
+  mov rcx,0x00
+  mov rdx,0x01
+  mov r8,r13
```

Comment:
If I'm not mistaken, this has the `hinstDLL` and `lpvReserved` arguments reversed. RCX should have the DLL handle / base address, while `r8` can be NULL for the reserved parameter.

Reply:
You'r right, my mistake :+1: